CheckoutTests/Tokenisation/TokenDetailsFactoryTests.swift

```diff
@@ -8,20 +8,81 @@
 import XCTest
 @testable import Checkout
 
-// swiftlint:disable implicitly_unwrapped_optional
 final class TokenDetailsFactoryTests: XCTestCase {
-  private var subject: TokenDetailsFactory! = TokenDetailsFactory()
-
-  override func tearDown() {
-    subject = nil
-
-    super.tearDown()
-  }
-
-  func test_create() {
-    let tokenResponse = StubProvider.createTokenResponse()
-    let expectedTokenDetails = StubProvider.createTokenDetails(expiryDate: ExpiryDate(month: 6, year: 2055))
-
-    XCTAssertEqual(subject.create(tokenResponse: tokenResponse), expectedTokenDetails)
-  }
+    
+    func testPropertiesMapped() {
+        let testType = TokenRequest.TokenType.card
+        let testToken = "12345676sdgdh"
+        let testExpiresOn = "tomorrow"
+        let testExpiryMonth = 6
+        let testExpiryYear = 3011
+        let testScheme = "visa"
```

Comment:
It can be anything, but this should be mada as well for proper readability of schemeLocal testing.

Reply:
I don't understand your suggestion. Similarly to [other response](https://github.com/checkout/frames-ios/pull/364#discussion_r1059341654), the goal of the test is to ensure a valid network object will be transformed successfully into an SDK model.
Not to validate all possible permutations.